src/main/java/org/springframework/data/redis/listener/RedisMessageListenerContainer.java

```diff
@@ -69,6 +70,7 @@
  * @author Way Joke
  * @author Thomas Darimont
  * @author Mark Paluch
+ * @author Jacques-Etienne Beaudet
```

Comment:
Let me know if my changes aren't worthy enough to be here, I don't really mind I was mainly following the contributions guideline!